data/pc/1.10/attributes.json

```diff
@@ -0,0 +1,67 @@
+[
+  {
+    "id": 1,
+    "name": "maxHealth",
```

Comment:
what's the difference between name and display name ? where is each used ?
do we want the default/min/max as well https://wiki.vg/Protocol#Entity_Properties ?
what is the id ?

in general: how will the data be used, is there enough info in that file to use it ?

Reply:
name is the name that prismarine-physics will call.
displayName is the name that minecraft uses.

Basically displayName can vary between versions but I want name to be consistant between versions.

```js
function getMojangAttribute (name) {
  return mcData.mojangAttribute(name) // returns displayName
}

getMojangAttribute('movementSpeed')
// returns generic.movementSpeed for pre 1.16
// returns minecraft:generic.movement_speed for 1.16+
```

Reply:
The id is what I copied from how the effects are used. I can remove it if that is needed.

Reply:
The protocol does not know names
I think maybe it knows ids. Is it these IDs?
That's what would be interesting to check

Reply:
Protocol gives mojangNames